### PR TITLE
Removed repeated exercise on prereq fenced div

### LINKED ISSUE #313: prereq exercise repeated twice
We ask learners to turn their prerequisites bullet point list into a prereq fenced div twice - in designing episode objectives and designing assessments.

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #313.

### CODE REVIEW

File: episodes/episode-objectives.md [hunk 1]
```diff
@@ -86,32 +86,6 @@ After completing this episode, participants should be able to...
 
 ::: challenge
```
Comment (tobyhodges) [inline episodes/episode-objectives.md:87]:
```suggestion
```

File: episodes/episode-objectives.md [hunk 2]
```diff
@@ -86,32 +86,6 @@ After completing this episode, participants should be able to...
...
-:::::::::::::::::
-```
-
-::::::
-
-:::
-
-::: callout
-
 ### Full List of Fenced Divs
```
Comment (tobyhodges) [inline episodes/episode-objectives.md:89]:
```suggestion
::: callout

### Full List of Fenced Divs
```

**Review by tobyhodges — APPROVED** [review]
I'm making a minor adjustment to avoid deleting the wrong fences, but otherwise this is good to merge. Thanks @anenadic 🙌

**Comment by anenadic** [comment]
> I'm making a minor adjustment to avoid deleting the wrong fences, but otherwise this is good to merge. Thanks @anenadic 🙌

Oh sorry about that - I made a mental note to go back to fix this then forgot. Good spot!

**Pull request merged**